db/migrate/20200220161956_add_equality_and_diversity_to_application_form.rb

```diff
@@ -0,0 +1,10 @@
+class AddEqualityAndDiversityToApplicationForm < ActiveRecord::Migration[6.0]
+  def change
+    change_table :application_forms do |t|
+      t.string :sex
+      t.string :disability
```

Comment:
This is a multiple choice with checkboxes, string would be a bit clunky to use I think. Should also probably be `disabilities`

Reply:
I'm wondering if it would be better to structure this data in the database using a JSON field, e.g.:

```json
{
  "sex": "male",
  "disabilities": ["blind", "deaf"],
  "ethnic_group": "asian",
  "ethnic_background": "chinese"
}
```

cc @tijmenb @duncanjbrown WDYT?

Reply:
I like the idea of using a JSON field